includes/class-wc-stripe-api.php

```diff
@@ -242,6 +263,18 @@ public static function retrieve( $api ) {
 			]
 		);
 
+		// If we get a 401 error, we know the secret key is not valid.
```

Comment:
Maybe we could consider later changing this to check for anything different from 200, to cover more possible issues.

Reply:
My proposed changes in #4327 take that approach when looking at responses. 😁